sphinx/tutorial/rose/suites.rst

```diff
@@ -533,7 +533,7 @@ Rose Applications In Rose Suite Configurations
 
    #. **Run The Suite.**
 
-      Install, validate and run and examine the suite::
+      Install, validate, run and examine the suite::
```

Comment:
If examining the suite, we need the `cylc gui` command back, no?

Reply:
It's now provided by cylc-uiserver.

Reply:
I've responded to this in https://github.com/metomi/rose/pull/2539/commits/1542ed7cd357d58fca42ce7dc9cc4346bdb39f0b